### ISSUE TITLE
svg.transform transform & translate not working (undesired units were added)

### ISSUE THREAD

**Issue by chengh42** [Zaid-Ajaj/Feliz#348]
Hi there,

Thanks for creating this fabulous package. The experience with Feliz was very enjoyable!

### Description of the issue

A minor issue I've encountered when using `svg.transform`, Feliz seems to be adding undesired units (here deg and px) that prevent the transform attribute from working. For instance, with:

```fsharp
Svg.svg [
    svg.g [
        svg.transform [ transform.rotate 11.11; transform.translate (-0.22, -0.33) ]
        svg.children [ ... ]
    ]
]
``` 

... the resulting svg elements unfortunately do not have the desired transform results:

```html
<svg>
    <g transform="rotate(11.11deg) translate(-0.22px, -0.33px)"> ... </g>
</svg>
```

### Known workaround

The workaround below does give the results as expected,

```fsharp
Svg.svg [
    svg.g [
        prop.custom ("transform", "rotate(11.11) translate(-0.22, -0.33)") :?> ISvgAttribute
        svg.children [ ... ]
    ]
]
```

... which has the resulting svg elements as followed,

```html
<svg>
    <g transform="rotate(11.11) translate(-0.22, -0.33)"> ... </g>
</svg>
```

... though I also understand that it's nicer to avoid using `prop.custom` :)

Was wondering if this is already fixed or maybe planned? As I see that there was a relevant fix on the removal of "px" for translate https://github.com/Zaid-Ajaj/Feliz/commit/5660fb5647e7d658215252771d0c8f18976e106e

Thanks again!

**Comment by Zaid-Ajaj** [comment]
Hi there @chengh42 thank you for the kind words 🙏 can you please tell me which version of Feliz you are using? 

We did indeed fix the issue with the unit in the `svg.translate` function and published a version back then but now looking at the code I see that we didn't remove the units from the `svg.rotate`. That said, the current `svg.transform` function will remove the units just in case in latest Feliz version so in theory this problem shouldn't occur if you are using latest Feliz version

**Comment by Zaid-Ajaj** [comment]
Published Feliz v1.41 which should now also remove the `deg` unit altogether. Can you please give it a try and let me know if it resolves your problem? thanks!

**Comment by chengh42** [comment]
Thanks for your extremely fast reply! Apologies for my delay.

I've tried using the latest release Feliz v1.42, unfortunately the problem remains 😢 The element using `svg.transform` is still adding `deg` and `px` to the rendered results. Here are the resulted screenshot and code snippet created using Feliz Template v3.9.0,

![image](https://user-images.githubusercontent.com/44002425/114736491-6c966800-9d46-11eb-8330-d68bce04e3e1.png)

```fsharp
namespace App

open Feliz
open Feliz.Router

type Switch =
    | NoTransform
    | SvgTransform
    | PropCustom

type Components =
    [<ReactComponent>]
    static member Needle (switch: Switch) = 
        Html.div [
            prop.style [ 
                // style.display.inlineBlock
                ]
            prop.children [
                Html.h3 [ 
                    match switch with
                        | NoTransform  -> prop.text "without transform"
                        | SvgTransform -> prop.text "svg.transform [ transform.rotate(-45); transform.translate (-0.2, -0.3) ]" 
                        | PropCustom   -> prop.text "prop.custom (\"transform\", \"rotate(-45) translate(-0.2, -0.3)\")"
                    ]
                Svg.svg [
                    svg.viewBox (-1, -1, 2, 2)
                    svg.width 200.
                    prop.style [ style.overflow.visible; style.border (1, borderStyle.solid, "black") ] :?> ISvgAttribute
                    svg.children [
                        Svg.g [
                            match switch with
                            | NoTransform  -> ()
                            | SvgTransform -> svg.transform [ transform.rotate(-45); transform.translate (-0.2, -0.3) ]
                            | PropCustom   -> prop.custom ("transform", "rotate(-45) translate(-0.2, -0.3)") :?> ISvgAttribute
                            svg.children [
                                Svg.path [
                                    svg.d "M0.136364 0.0290102 C0.158279 -0.0096701 0.219156 -0.00967009 0.241071 0.0290102C0.297078 0.120023 0.375 0.263367 0.375 0.324801C0.375 0.422639 0.292208 0.5 0.1875 0.5C0.0852272 0.5 -1.8346e-08 0.422639 -9.79274e-09 0.324801C0.00243506 0.263367 0.0803571 0.120023 0.136364 0.0290102ZM0.1875 0.381684C0.221591 0.381684 0.248377 0.356655 0.248377 0.324801C0.248377 0.292947 0.221591 0.267918 0.1875 0.267918C0.153409 0.267918 0.126623 0.292947 0.126623 0.324801C0.126623 0.356655 0.155844 0.381684 0.1875 0.381684Z"
                                    svg.fill "#black"
                                ]
                            ]
                        ]
                    ]
                ]
            ]
        ]

    [<ReactComponent>]
    static member Router() =
        let (currentUrl, updateUrl) = React.useState(Router.currentUrl())
        React.router [
            router.onUrlChanged updateUrl
            router.children [
                match currentUrl with
                | [ ] -> 
                    Html.div [
                        Html.h1 "Samples"
                        Html.div [ 
                            Components.Needle NoTransform
                            Components.Needle SvgTransform
                            Components.Needle PropCustom
                            ]
                    ]
                | [ "needle-svg-transform" ] -> Components.Needle SvgTransform
                | [ "needle-prop-custom" ]   -> Components.Needle PropCustom
                | otherwise -> Html.h1 "Not found"
            ]
        ]
```

**Comment by Zaid-Ajaj** [comment]
Thanks a lot for the repro code. I'll have another look and let you know how it goes 😉

**Comment by Zaid-Ajaj** [comment]
Fixed now, should be working as of v1.43 please give it a try and let me know how it goes. Thanks again for the repro

**Comment by chengh42** [comment]
Yes beautiful!! Thanks a lot for looking into this 😃
